Assets/MixedRealityToolkit.Tests/EditModeTests/InputSystem/TestFixture_02_InteractionDefinitionTests.cs

```diff
@@ -138,7 +138,13 @@ public void Test04_TestBoolNoChange()
 
             interaction.BoolData = testValue;
```

Comment:
```suggestion
            interaction.BoolData = true;
```

Reply:
There's not a lot of value having a bool that is assigned to true, and used whereever the 'true' value is desired - just use true directly (same goes for other parts of this file)